### PR TITLE
Every time a map receives a like the owner receives a notification

### PR DESCRIPTION
**Pull request by ethervoid**
When the map of a user receives a like, he/she is going to receive a notification.

This solves #1410

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5556/))

**Comment by saleiva** [comment]
LTMK when deploying this! :)

**Comment by ethervoid** [comment]
Please @Kartones check this PR pls

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5571/))

File: lib/resque/user_jobs.rb [hunk 1]
```diff
@@ -97,12 +97,22 @@ module UnshareTable
...
           UserMailer.unshare_table(table_name, table_owner_name, u).deliver
         end
       end
 
+      module MapLiked
+        extend ::Resque::Metrics
+        @queue = :users
+
+        def self.perform(visualization_id, viewer_user_id)
+          viz = CartoDB::Visualization::Member.new(id: visualization_id).fetch
```
Comment (Kartones) [inline lib/resque/user_jobs.rb]: Uhm, could you try using the new models? `Carto::Visualization` and `Carto::User`
We should stop using old ones except where it is critical. e.g. writes currently are not migrated, but reading fields/properties like probably this case falls into should be done using new models.

And if a new model is missing some "read method", the idea is to migrate it from old one.

**Comment by Kartones** [comment]
Please change to use new models (I think will work without any issue), except for that +1

**Comment by ethervoid** [comment]
Done, waiting for a change to the email template by @matallo and :shipit:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5581/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5584/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5588/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5591/))

**Pull request merged**